lib/vquic/curl_osslq.c

```diff
@@ -1239,6 +1239,29 @@ static const struct alpn_spec ALPN_SPEC_H3 = {
   SSL_set_bio(ctx->tls.ossl.ssl, bio, bio);
   bio = NULL;
   SSL_set_connect_state(ctx->tls.ossl.ssl);
+
+  if(data->set.quic_version == 2) {
+    uint32_t quic_versions[] = { 0x6b3343cf }; /* QUIC v2 */
+    size_t num_versions = sizeof(quic_versions) / sizeof(quic_versions[0]);
+    if(!SSL_set_quic_transport_versions(ctx->tls.ossl.ssl, quic_versions,
+                                       num_versions)) {
+      failf(data, "Failed to set QUIC v2 transport version");
+      result = CURLE_QUIC_CONNECT_ERROR;
+      goto out;
+    }
+  }
+  else if(data->set.quic_version == 1) {
+    uint32_t quic_versions[] = { 0x00000001 }; /* QUIC v1 */
+    size_t num_versions = sizeof(quic_versions) / sizeof(quic_versions[0]);
+    if(!SSL_set_quic_transport_versions(ctx->tls.ossl.ssl, quic_versions,
+                                       num_versions)) {
```

Comment:
```suggestion
    if(!SSL_set_quic_transport_versions(ctx->tls.ossl.ssl,
                                        CURL_ARRAYSIZE(quic_versions),
                                        num_versions)) {
```
Let's use CURL_ARRAYSIZE() here too, and fix indentation.